src/main/java/com/idrsolutions/microservice/BaseServletContextListener.java

```diff
@@ -200,7 +230,7 @@ private void validateInputPath(final Properties properties) {
     private void validateOutputPath(final Properties properties) {
         final String outputPath = properties.getProperty(KEY_PROPERTY_OUTPUT_PATH);
         if (outputPath == null || outputPath.isEmpty()) {
-            final String outputDir = getConfigPath() + "output";
+            final String outputDir = getConfigPath() + "output/";
```

Comment:
Is there a specific reason why the '/' is added? Why is it only applied to the output path and not also the input path?